backend/schedulerd/cron_scheduler.go

```diff
@@ -101,9 +105,10 @@ func (s *CronScheduler) Interrupt(check *corev2.CheckConfig) {
 
 // Stop stops the cron scheduler.
 func (s *CronScheduler) Stop() error {
+	s.cancel()
+	s.stopWg.Wait()
 	cronCounter.WithLabelValues(s.check.Namespace).Dec()
 	logger.Info("stopping cron scheduler")
```

Comment:
Does it make more sense to move this log line above the call to `s.stopWg.Wait()` similar to the `Stop()` method in `IntervalScheduler`?